migration-tool/src/main/java/software/amazon/awssdk/migration/recipe/V1BuilderVariationsToV2Builder.java

```diff
@@ -134,9 +141,11 @@ private J.MethodInvocation renameStandardToBuilder(J.MethodInvocation method, Ja
                 Collections.emptyList()
             );
 
+            Space spacing = "builder".equals(methodName) ? Space.EMPTY : Space.SINGLE_SPACE;
```

Comment:
We are swapping return type `SqsClientBuilder` for `SqsClient` [here](https://github.com/aws/aws-sdk-java-v2/blob/feature/master/migration-tool/migration-tool/src/main/java/software/amazon/awssdk/migration/recipe/V1BuilderVariationsToV2Builder.java#L104) so the right class is being returned for `create()`

The spacing is needed, currently we depend on the spacing from `build()`, the transform on 
`AmazonSQSClientBuilder sqs = AmazonSQSClientBuilder.standard();`
becomes
`SqsClientBuilder sqs =SqsClient.builder();`

Can address in separate PR, wdyt?

Reply:
>We are swapping return type SqsClientBuilder for SqsClient [here](https://github.com/aws/aws-sdk-java-v2/blob/feature/master/migration-tool/migration-tool/src/main/java/software/amazon/awssdk/migration/recipe/V1BuilderVariationsToV2Builder.java#L104) so the right class is being returned for create()

That seems to be a bug. `methodType` should return builder type.

```
            JavaType.Method methodType = new JavaType.Method(
                null,
                0L,
                v2Client,
                methodName,
                fullyQualified,
                Collections.emptyList(),
                Collections.emptyList(),
                Collections.emptyList(),
                Collections.emptyList()
            );
```

Instead of hardcoding single space, can we get it from the method?

```
            J.MethodInvocation builderInvoke = new J.MethodInvocation(
                Tree.randomId(),
                method.getPrefix(),
                Markers.EMPTY,
                JRightPadded.build(id),
                null,
                builderMethod,
                JContainer.empty(),
                methodType
            );
```

Reply:
updated return type based on method, and spacing to use `method.getPrefix()`